frontend/src/metabase/query_builder/components/MoveQuestionModal/MoveQuestionModal.tsx

```diff
@@ -211,9 +211,17 @@ export const MoveQuestionModal = ({
           onClose={onClose}
           title={
             <Title fz="1.25rem" lh={1.5}>
-              Moving this question to another dashboard will remove it from{" "}
-              <Icon name="dashboard" style={{ marginBottom: -2 }} size={20} />{" "}
-              <Dashboards.Name id={question.dashboardId()} />
+              {c("{0} is the name of a dashboard")
+                .jt`Moving this question to another dashboard will remove it from ${(
+                <>
+                  <Icon
```

Comment:
prettier? This looks a little different than the master one.
![image](https://github.com/user-attachments/assets/40aab8ba-a116-4e28-9418-e6d10c2e1b4d)